capability-provider/Cross.toml

```diff
@@ -12,8 +12,3 @@ image = "wasmcloud/cross:aarch64-apple-darwin"
 
 [target.x86_64-unknown-linux-gnu]
 image = "wasmcloud/cross:x86_64-unknown-linux-gnu"
-
-[build.env]
-passthrough = [
-    "XDG_CACHE_HOME",
```

Comment:
I think this might still be needed?

Reply:
I don't see it on other repos that work. But I'm just wildly guessing at this point

Reply:
I see it on blobstore-fs, blobstore-s3, kv-vault, and direct-dispatch. I don't _not_ see it on any other provider